axlearn/audio/asr_decoder.py

```diff
@@ -28,6 +28,39 @@
 from axlearn.common.utils import Nested, Tensor
 
 
+def is_valid_ctc_seq(logitpaddings, labels, labelpaddings):
+    """Returns for per example sequence if it passes validity check.
+
+    Note that the above `ctc_loss_with_alignments` returns logeps
```

Comment:
What is `ctc_loss_with_alignments` referring to?

Reply:
hmm that seems to be a follow up PR.